### PR TITLE
[For Review] Added a sample Popup

### PR DESCRIPTION
**Pull request by shaunanoordin**
## PR Oveview
* This PR is for @wgranger to review; the important thing is sharing some functional popup modal code, which can then be updated as necessary.
* In this PR, we have an example of how we can create a simple "message popup" for the project.
* The `<Popup>` component is a simple container and can be placed in any container/component you want.
  * Clicking on either the 'close' cutton or on the black background will close the popup.
  * **Of note,** the show/hide logic of the Popup container is actually handled on the parent container/component.
* The style is up for debate and has plenty of room for improvement. E.g. make it fit content instead of trying to fit the screen.
* Possible bug?: the popup _doesn't steal events_ when active, so you can technically still mouse-scroll the background items with the popup open.

### CODE REVIEW

File: src/components/Popup.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  close(e) {
+    this.props.onClose && this.props.onClose();
+    return this.stopEvent(e);
+  }
+  
+  stopEvent(e) {
+    e.preventDefault && e.preventDefault();
+    e.stopPropagation && e.stopPropagation();
+    e.returnValue = false;
+    e.cancelBubble = true;
```
Comment (wgranger) [inline src/components/Popup.jsx]: This is a good opportunity to review these event functions.

**Comment by wgranger** [comment]
I'm going to go ahead and merge this PR, since the main functionality is present and I can begin work on the components of the app that need a popup. I'm also pinging @beckyrother for some design input on the popup.

screen shot 2017-07-24 at 3 27 36 pm

**Comment by beckyrother** [comment]
Thanks @wgranger do you have some time tomorrow to sit down and look at this?

**Comment by wgranger** [comment]
@beckyrother For sure

**Pull request merged**